### ISSUE TITLE
Distinguish the footer from the content

### ISSUE THREAD

**Issue by ru-fu** [evildmp/vanilla-sphinx-test#28]
Currently, the footer looks like it's part of the content. It should be distinguished in some way (maybe a line before the footer, or a different font size/color).

![image](https://user-images.githubusercontent.com/11227796/143554745-a8776b26-f23a-43b8-9f66-e27ceac89d0e.png)

**Comment by evildmp** [comment]
Each Canonical product website has its own footer (compare https://anbox-cloud.io/ with https://juju.is/docs/olm and https://maas.io/ for example).

Ideally, the documentation should in each case adopt the website's footer. Maybe it's possible to break that out into a `footer.html` so it's easily adapted on a case-by-case basis?

**Comment by ru-fu** [comment]
True, we should probably have a product-specific footer in the same way we need a product-specific header.

The information I find most relevant in the Sphinx footer is the build date and the "Show source" though, and I'm not sure I would want those at the very bottom of the page (especially if the page footer is quite big). So maybe we need a Sphinx footer and a page footer?

**Comment by ru-fu** [comment]
CSS override that fixes the Sphinx footer:
```
/** Fix for issue #28 **/
footer {
    color: #666
}

footer::before {
    border-top: 1px #ebebeb solid;
    content: "";
    display: inherit;
    padding-bottom: 10px;
}
```
